lesson_07/objects/objects_app/src/test/java/com/codedifferently/lesson7/MohamedTest.java

```diff
@@ -0,0 +1,60 @@
+package com.codedifferently.lesson7;
+
+// import java.lang.reflect.Method;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import com.codedifferently.lesson7.MohamedObjects.Main.CPU;
+import com.codedifferently.lesson7.MohamedObjects.Main.GPU;
+import com.codedifferently.lesson7.MohamedObjects.Main.IllegalYearExeption;
+import com.codedifferently.lesson7.MohamedObjects.Main.cpuBrands;
+import com.codedifferently.lesson7.MohamedObjects.Main.portTypes;
+import java.util.*;
+import org.junit.jupiter.api.Test;
+
+class MohamedTest {
+
+  @Test
+  public void setClockSpeed() {
+    CPU cpu = new CPU("cpu1", 2000, cpuBrands.INTEL, 23.32);
+    assertEquals(23.32, cpu.getClockSpeed());
+  }
+
+  @Test
+  void gpuReleaseYear() throws IllegalYearExeption {
+    GPU aGpu = new GPU();
+    try {
```

Comment:
use `assertThatThrownBy` to test the instance and message type for the expected exception. Also, come up with a better name for your test.